Source/OxyPlot/Rendering/ScreenVector.cs

```diff
@@ -134,6 +134,9 @@ public double Y
         /// <summary>
         /// Normalizes this vector.
         /// </summary>
+        /// <remarks>
+        /// If the vector has zero length, then it is left unchanged.
+        /// </remarks>
         public void Normalize()
```

Comment:
I see that there is some precedent for the mutable `struct` stuff, and also for pretty much all the other stuff I didn't like in `DataVector`... And unfortunately, changing `ScreenVector.Normalize` to be non-mutable would be a breaking change... But maybe one we would be willing to take?

Reply:
Removed this method and make `x` and `y` readonly for the time being so there's no risk of not getting around to that once this goes in.